app/views/admin/questions/index.html.slim

```diff
@@ -4,12 +4,16 @@
 	= link_to 'Create new question', new_admin_question_path, class: 'btn btn-primary btn-question'
 	.center
 		= will_paginate
-	table
-	  tr
-	    th
-	      | Content
-	    th
-	      | Text
-	  = render @questions
-	.center
-		= will_paginate
+    h3 
+      | &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Topic&nbsp;&nbsp;&nbsp;Content
+      - params[:page] = 1 if params[:page].to_i == 0
+      ol start=("#{( ( params[:page].to_i * 10 ) - 9 )}") 
```

Comment:
The idea was to use ```question.id```.
Idea with ```ol``` is pretty clever, but still please use ```question.id``` as ordinal number.